prime-dotnet-webapi/Views/Emails/RemoteUsersUpdatedEmail.cshtml

```diff
@@ -0,0 +1,28 @@
+@using Prime.ViewModels.Emails
+@model RemoteUsersUpdatedEmailViewModel
+
+<p>
+  Notification: The list of Remote Practitioners at @Model.SiteStreetAddress of
+  @Model.OrganizationName (PEC: @Model.SitePec) has been updated. The Remote Practitioners
```

Comment:
Can `@Model.SitePec` be null?  

![image](https://user-images.githubusercontent.com/7586665/106486044-7a8a5b80-646e-11eb-92eb-4d9d3978b257.png)